tests/dummy/app/components/configurable-table-source.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
do you really want to make this a component? it's not needed after the changes on master